tcti/tcti.h

```diff
@@ -53,7 +53,7 @@
 #define TCTI_LOG_DATA(ctx)     ((TSS2_TCTI_CONTEXT_INTEL*)ctx)->logData
 #define TCTI_LOG_BUFFER_CALLBACK(ctx) ((TSS2_TCTI_CONTEXT_INTEL*)ctx)->logBufferCallback
 #define TCTI_CONTEXT ((TSS2_TCTI_CONTEXT_COMMON_CURRENT *)(SYS_CONTEXT->tctiContext))
-#define TCTI_CONTEXT_INTEL ((TSS2_TCTI_CONTEXT_INTEL *)tctiContext)
+#define TCTI_CONTEXT_INTEL(ctx) ((TSS2_TCTI_CONTEXT_INTEL*)ctx)
```

Comment:
Why not use static inlines, same code as a macro *WITH* type checking.

Reply:
Something like `static inline TSS2_TCTI_CONTEXT_INTEL* cast_function (TSS2_TCTI_CONTEXT* ctx)`

Reply:
Yep....